devices/libsdio/README.md

```diff
@@ -0,0 +1,277 @@
+# libsdio
+
+## Description
+
+libsdio is a static, precompiled library containing a generic SDIO driver which directly controls platform hardware. This driver defines an API providing basic interface control functionality.
```

Comment:
```suggestion
libsdio is a static, precompiled library containing a generic SDIO driver which directly controls the platform hardware. This driver defines an API providing basic an interface control functionality.
```

Reply:
`basic an interface`? Presuming you meant to insert `an` before `basic`, in which case `a` should be used due to the following word beginning with a phonetic consonant. Changing to `a basic interface control functionality`.

Reply:
I agree with you, my mistake